cmd/config-current.go

```diff
@@ -817,7 +817,10 @@ func GetHelp(subSys, key string, envOnly bool) (Help, error) {
 }
 
 func newServerConfig() config.Config {
-	return config.New()
+	srvCfg := config.New()
+	// For fresh installs, apply secure API defaults
+	applySECUREAPIDefaults(srvCfg)
```

Comment:
```suggestion
	applyFreshInstallAPIDefaults(srvCfg)
```

`SECURE` in all-caps is against the convention for Go function names, but in any case I think this could be more generic and just refer to any setting that we want to apply on all new fresh installations.